src/index.ts

```diff
@@ -239,7 +248,7 @@ export async function generate(options: GenerateOptions, logger: Logger = CLI_LO
             if (err.code !== 'ENOENT') {
                 throw err
             }
-            logger.error(chalk.red('\nCould not find template ' + options.template + '\n'))
+            throw new TemplateNotFoundError(options.template)
```

Comment:
Why was this not terminating before but is now?

Reply:
This actually changes the current behavior, but previously, when the template was not found, we just created an empty file, which is confusing imo. If you specifically set the template and its not found, it should fail, what do you think?